cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -961,3 +972,95 @@ func (ht *hcsTask) ProcessorInfo(ctx context.Context) (*processorInfo, error) {
 		count: ht.host.ProcessorCount(),
 	}, nil
 }
+
+// Add mount as vSMB share to the UVM at the given destination path
```

Comment:
will we also want to support hot adding scsi devices in the future?

Reply:
We could if there is a request for it in the future.  Only this function would have to be extended to check for the incoming mount type and call appropriate functions for that particular mount type

Reply:
scsci mounts can have specific naming conventions which can be used to extend the behavior in hcsshim